commands/auto.js

```diff
@@ -1,7 +1,7 @@
-const channels = require('../index').channels,
-  getText = require('../utils/getText'),
-  { addThread, removeThread } = require('../utils/threadActions'),
-  { CommandInteraction, Permissions } = require('discord.js');
+const channels = require('../index').channels;
+const getText = require('../utils/getText');
+const { addThread, removeThread } = require('../utils/threadActions'),
```

Comment:
this would fail due to the `,` at the end of the variable assignment. I see no reason to put const before every keyword as it is functionally the same